versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 4.0"
 
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.6.0"
+      version = ">= 4.60.0"
```

Comment:
Please use a constraint that keeps the same major AWS provider version, like `version = "~> 4.0"`

Reply:
Fixed